src/graphics/device.js

```diff
@@ -685,7 +685,7 @@ var GraphicsDevice = function (canvas, options) {
     this._spectorCurrentMarker = "";
     // #endif
 
-    this.createGrabPass();
+    this.createGrabPass(options.alpha, options.antialias);
```

Comment:
So, I might be wrong here, but isn't it the case that a WebGL implementation is not required to honor the context creation options? So you could *request* alpha, but not necessarily get it? Same deal with antialising.

Reply:
That is possible. What other way to detect this is there? These functions are not supported on all platforms:
https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/getFramebufferAttachmentParameter
https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/getRenderbufferParameter

Reply:
Hey @kenrussell, do you have any tips for this?

Reply:
Per the spec, https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.2.1 , alpha is required to be honored exactly as specified. antialias is essentially a hint, and might not be honored.

I'm not sure exactly what information you need but you can figure out whether antialiasing was enabled for the back buffer by querying the SAMPLES parameter:

<pre>
let c = document.createElement('canvas');
let gl = c.getContext('webgl');
gl.getParameter(gl.SAMPLES)
> 4

let c2 = document.createElement('canvas');
let gl2 = c2.getContext('webgl', { antialias: false });
gl2.getParameter(gl.SAMPLES)
> 0
</pre>

If there seems to be some bug in Chromium with the combination of antialias:true and alpha:false, please provide a small test case and file a bug on crbug.com . Please include about:gpu from your system when doing so, and ping me the bug ID. Thanks.

Reply:
Thanks for the words of wisdom @kenrussell! Much appreciated. 😄

Reply:
@kenrussell - thanks for the info, that was helpful.
I have created an issue now - only can reproduce in Chrome on Mac, there was similar issue in the past already, see the description: https://bugs.chromium.org/p/chromium/issues/detail?id=1156756